crates/kobold/Cargo.toml

```
@@ -16,7 +16,7 @@ wasm-bindgen = "0.2.84"
 wasm-bindgen-futures = "0.4.34"
 itoa = "1.0.1"
 ryu = "1.0.12"
-kobold_macros = "0.5.0"
+kobold_macros = { path = "../kobold_macros", version = "0.5.0" }
```

I like this order better, that seems to be a semi-standard too looking at some high-quality popular crates.

```suggestion
kobold_macros = { version = "0.5.0", path = "../kobold_macros" }
```